app/src/main/java/com/teamagam/gimelgimel/app/message/viewModel/SendSimpleMessageViewModel.java

```diff
@@ -0,0 +1,12 @@
+package com.teamagam.gimelgimel.app.message.viewModel;
+
+import javax.inject.Inject;
+
+
+public class SendSimpleMessageViewModel extends SendMessageViewModel {
```

Comment:
You should delete this class, you've created it because of the name.
You have the functionality you want in the base class, so use it instead.